src/main/java/com/xiaomi/infra/pegasus/rpc/interceptor/InterceptorManger.java

```diff
@@ -1,18 +1,22 @@
 package com.xiaomi.infra.pegasus.rpc.interceptor;
 
 import com.xiaomi.infra.pegasus.base.error_code.error_types;
+import com.xiaomi.infra.pegasus.client.PException;
 import com.xiaomi.infra.pegasus.rpc.TableOptions;
 import com.xiaomi.infra.pegasus.rpc.async.ClientRequestRound;
 import com.xiaomi.infra.pegasus.rpc.async.TableHandler;
 import java.util.ArrayList;
 import java.util.List;
+import org.slf4j.Logger;
 
 public class InterceptorManger {
+  private static final Logger logger = org.slf4j.LoggerFactory.getLogger(InterceptorManger.class);
 
   private List<TableInterceptor> interceptors = new ArrayList<>();
 
   public InterceptorManger(TableOptions options) {
-    register(new BackupRequestInterceptor(options.enableBackupRequest()));
+    this.register(new BackupRequestInterceptor(options.enableBackupRequest()))
```

Comment:
```suggestion
  if(options.enableBackupRequest()) {
    this.register(new BackupRequestInterceptor(options.enableBackupRequest()))
  }
```

No need for the `isOpen` mark in each interceptor.